### ISSUE TITLE
Template encoding not ok on CC header

### ISSUE THREAD

**Issue by jorgebnunes** [thundernest/quicktext#353]
Hi!

First of all, thank you for your work!
Regarding this problem, if I place this on templaqte text:

[[HEADER=cc|Sónia ...]], which is equivalent HTML to: S\&oacute;nia ... , on CC it renders like this:

S�nia ...

Please verify this.
Thanks.

**Comment by SamuelPlentz** [comment]
I can confirm that:
```
[[HEADER=cc|Sónia@example.com]]
[[HEADER=cc|Müller@Beispiel.de]]
```
get scrambled to ```S�nia@example.com``` and ```M�ller@Beispiel.de```.

However without using the Tag ```HEADER``` the text
```
Sónia@example.com
Müller@Beispiel.de
```
gets inserted correctly.

**Comment by jobisoft** [comment]
Please verify:
https://github.com/jobisoft/quicktext/releases/tag/v5.6

**Comment by SamuelPlentz** [comment]
I can confirm, that this is fixed in version 5.6.
